### ISSUE TITLE
Consider randomizing the User-Agent being passed in the base-header

### ISSUE THREAD

**Issue by ashaychangwani** [pallupz/covid-vaccine-booking#133]
In my experience, repeated use of the same user agent header has lead to far more non-200 status code responses.

You can use this library called fake_useragent as:
from fake_useragent import UserAgent
user_agent = UserAgent()
HEADER['User-Agent'] = user_agent.random

Just a suggestion!

**Comment by ashaychangwani** [comment]
PR incoming :D

**Comment by pallupz** [comment]
I guess you'd have to modify only ```src/covid-vaccine-slot-booking.py ```

**Comment by ashaychangwani** [comment]
Yeah I only need to modify that file, but I can't push my commit. 
![image](https://user-images.githubusercontent.com/29693329/117477549-c112b900-af7b-11eb-89d6-fb114a47127f.png)

**Comment by vishalv26** [comment]
@ashaychangwani please fork the repo and send PR

**Comment by ashaychangwani** [comment]
@pallupz do I need to fork the repo for a 5 line PR?

**Comment by pallupz** [comment]
link to discord server if anyone's interested in a single place for discussions : https://discord.gg/rCz8VdRB

**Comment by vishalv26** [comment]
@pallupz now someone will come and say we should implement tor to get new ip's every now and then. what is your view on this 😕

**Comment by pallupz** [comment]
That would be something. I dont know enough about it to comment authoritatively

**Comment by pallupz** [comment]
link to discord server if anyone's interested in a single place for discussions : https://discord.gg/rCz8VdRB
